packages/turf-unkink-polygon/lib/simplepolygon.ts

```diff
@@ -563,23 +634,20 @@ function equalArrays(array1, array2) {
 }
 
 // Fix Javascript modulo for negative number. From http://stackoverflow.com/questions/4467539/javascript-modulo-not-behaving
-function modulo(n, m) {
+function modulo(n: number, m: number) {
   return ((n % m) + m) % m;
 }
 
 // Function to check if array is unique (i.e. all unique elements, i.e. no duplicate elements)
-function isUnique(array) {
-  var u = {};
+function isUnique(array: Position[]) {
+  var u: Record<string, number> = {};
   var isUnique = 1;
   for (var i = 0, l = array.length; i < l; ++i) {
-    if (Object.prototype.hasOwnProperty.call(u, array[i])) {
+    if (Object.prototype.hasOwnProperty.call(u, array[i].toString())) {
       isUnique = 0;
       break;
     }
-    u[array[i]] = 1;
+    u[array[i].toString()] = 1;
```

Comment:
This is probably horrible for perf `[number, number]`.toString() gets used as the key into `u`